### PR TITLE
Update citation for accepted mito paper

### LINKED ISSUE #877: Add published gnomAD mito paper to browser "Publications" page
Paper has been accepted in principle and we are waiting on proofs and a publication date!

### PR DESCRIPTION
**Pull request by phildarnowsky-broad**
Resolves #877

### CODE REVIEW

**Comment by phildarnowsky-broad** [comment]
@mattsolo1 Ready for a look.

Could you also remind me please, who else is good to get review from, and what the process is once the reviewer approves?

**Review by mattsolo1 — APPROVED** [review]

**Comment by mattsolo1** [comment]
I'm probably the best person to review most PRs at the moment. In some cases you could try asking people who filed the issue, especially if they are on comp bios on the methods team.

After approval you can go ahead with rebase and merge.

**Comment by phildarnowsky-broad** [comment]
@mattsolo1 great, thanks

**Pull request merged**
